src/collectors/ceph/ceph.py

```diff
@@ -12,22 +12,32 @@
 
 """
 
-try:
-    import json
-except ImportError:
-    import simplejson as json
-
+import json  # No need for simplejson fallback b/c ceph py modules are >=2.6
 import glob
 import os
 import subprocess
-
+import re
 import diamond.collector
+import diamond.convertor
+from diamond.collector import str_to_bool
+
+
+# Metric name/path separator
+_PATH_SEP = "."
```

Comment:
@jaingaurav I am happy to change this and I'm not quite sure I understand your concern here. _PATH_SEP is "private" to this module and currently set to the same token other collectors use. It's only really a problem if someone goes and changes it. Are you suggesting that I pull this into a separate PR and we can discuss it later? I don't really have strong feelings one way or the other.